### PR TITLE
Hostname Page

### PR DESCRIPTION
**Pull request by reaganlo**
Introduces new hostname page in installer gui

### CODE REVIEW

File: gui/pages/hostname.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
+// Copyright © 2018 Intel Corporation
```
Comment (mdhorn) [inline gui/pages/hostname.go]: Please set the copyright to 2019

File: gui/pages/hostname.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package pages
+
+import (
+	"github.com/clearlinux/clr-installer/gui/utils"
+	"github.com/clearlinux/clr-installer/hostname"
+	"github.com/clearlinux/clr-installer/model"
+	"github.com/gotk3/gotk3/gtk"
```
Comment (mdhorn) [inline gui/pages/hostname.go]: Please move the 3rd party gtk package above our packages separated by a blank line.

Our import policy has been: 
- Core golang modules (no path0
- 3rd party (include path)
- local code

File: gui/pages/hostname.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+		model:      model,
+	}
+	var err error
+
+	page.box, err = utils.SetupBox(gtk.ORIENTATION_VERTICAL, 0, 8)
+	if err != nil {
+		return nil, err
+	}
+
+	// Hostname entry
```
Comment (mdhorn) [inline gui/pages/hostname.go]: Can we add a label before the entry box: "Hostname:"?

File: gui/pages/hostname.go [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+		return nil, err
+	}
+	page.HostnameWarning.SetXAlign(0)
+	page.HostnameWarning.SetYAlign(0)
+	sc, err := page.HostnameWarning.GetStyleContext()
+	if err != nil {
+		return nil, err
+	}
+	sc.AddClass("installer-warning-msg")
+	page.box.PackStart(page.HostnameWarning, true, true, 0)
```
Comment (mdhorn) [inline gui/pages/hostname.go]: We should add some padding between entry box and the warning label.

**Comment by mdhorn** [comment]
@reaganlo I assume this is replaced by #271 ? If so, please close.

**Comment by reaganlo** [comment]
Closing this PR because this is taken care of by #271

**Pull request closed without merging**